modules/con_enabling_annotation_based_resource_tracking_in_argo_cd.adoc

```diff
@@ -0,0 +1,143 @@
+----
++
+<1> Specifies the name of the second Argo CD instance.
+<2> Specifies the namespace used for the second Argo CD instance.
+<3> Specifies the name of the `installationID` for the second Argo CD instance.
++
+. Configure and label target namespaces to associate namespaces with their Argo CD instances.
+.. Navigate to *Administration* -> *Namespaces*.
+.. Create namespaces for application deployments, `app-ns-1` and `app-ns-2`.
+.. To associate each namespace with their respective Argo CD instances, run the following command:
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 argocd.argoproj.io/managed-by=argocd-test-demo-1
+$ oc label namespace app-ns-2 argocd.argoproj.io/managed-by=argocd-test-demo-2
+----
++
+. Create two applications in Argo CD.
+.. In the OpenShift Web Console, go to *Operators* → *Installed Operators* → *OpenShift GitOps Operator*.
+.. Select *Argo CD* and navigate to the *Applications* tab.
+.. Click *Create Application*.
+.. Enter the following YAML snippet to create two applications in Argo CD.
++
+.Example first application using Argo CD
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sprint-petclinic # <1>
+  namespace: argocd-test-demo-1 # <2>
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/redhat-developer/gitops-operator
+    path: test/examples/nginx
+    targetRevision: HEAD
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: app-ns-1
+  syncPolicy:
+    automated: {}
+----
+<1> Specifies the name of the first application.
+<2> Specifies the namespace used for the first application.
++
+.Example second application using Argo CD
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sprint-petclinic # <1>
+  namespace: argocd-test-demo-2 # <2>
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/redhat-developer/gitops-operator
+    path: test/examples/nginx
+    targetRevision: HEAD
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: app-ns-2
+  syncPolicy:
+    automated: {}
+----
+<1> Specifies the name of the second application that is created with the same name as the first application.
+<2> Specifies the namespace used for the second application.
+
+. Next, verify the deployment of each application in the {OCP} Web console.
+.. Navigate to *Workloads* → *Pods*.
+.. Ensure that the pods for Argo CD instances, `argocd-instance-demo-1` and `argocd-instance-demo-2` are running.
+.. Check the application synchronization status in the *Argo CD Applications* tab.
+.. Navigate to *Logs* under each Argo CD instance and confirm if you can view the status *Running in namespace-scoped mode*.
++
+A successful status message, *Running in namespace-scoped mode*, appears in each Argo CD instance. This status indicates that multiple applications with the same name are successfully deployed in the Argo CD instance.
```

Comment:
You can remove this line as there is no such message expected to appear in Argo CD instance